app/src/main/java/com/kickstarter/ui/views/compose/checkout/CollectionPlan.kt

```diff
@@ -219,33 +167,41 @@ fun PledgeOption(
                     style = typographyV2.subHeadlineMedium,
                     color = if (isSelectable) colors.textPrimary else colors.textDisabled
                 )
-                if (showBadge) {
-                    Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+
+                Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+
+                if (showBadge && plotMinimum != null) {
                     PledgeBadge(plotMinimum = plotMinimum)
-                } else if (description != null) {
-                    Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+                } else if (!description.isNullOrEmpty()) {
                     Text(
                         modifier = Modifier
-                            .padding(bottom = dimensions.paddingMedium)
+                            .padding(bottom = dimensions.paddingSmall)
```

Comment:
please revert this back to `paddingMedium`, as the padding on the bottom of the collection plan is too small